recipes/symstore/meta.yaml

```
@@ -0,0 +1,40 @@
+{% set name = "symstore" %}
+{% set version = "0.2.6" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: "f49ac8031b7572b5f3bcac3a8296d498fcd2a410225b3886f1c6b90b21c5bbf0"
```

Could you please remove the quotes from ``sha26``?
```suggestion
  sha256: f49ac8031b7572b5f3bcac3a8296d498fcd2a410225b3886f1c6b90b21c5bbf0
```

---

dropped the ""